logo.svg

```diff
@@ -0,0 +1,36 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 140">
```

Comment:
I'd like to keep the logo external to this repository, we have a dedicated repository for logo and other images